### DISCUSSION TITLE
Welcome to CountryCodePickerProject Discussions!

### DISCUSSION THREAD

**Discussion by hbb20** [hbb20/CountryCodePickerProject#456] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AminFarhat642** [comment]
how is setDefaultCountryUsingNameCode() method used?

**Reply by hbb20** [reply]
setDefaultCountryUsingNameCode() will take alpha 2 code of country and make that country as default country which will be a fallback country if detection is failed or anything goes wrong. But calling `setDefaultCountryUsingNameCode()` will not set it as selected country. To set that country as selected country, you will have to call `resetToDefaultCountry()` immediately after that.

**Comment by astankov** [comment]
'Is there posibility to change text "select a country" to "select a country or region"
![image](https://user-images.githubusercontent.com/1186218/114999601-4be32500-9ea2-11eb-9e24-61660eebc0c7.png).
I got rejected from AppGallery store bescause ot that.
Rejections description:
[Description]: We found that it does not display correctly when selecting country during login/registration function in your app (for example, some of the regions should not be shown as countries). Please see the attachment. Please refer to the rule 4.9 of "AppGallery Review Guidelines": https://developer.huawei.com/consumer/en/doc/distribution/app/30202 [Suggestion]: Please change [Select a country] to [Select a country / region], and rule out such problems under other modules or other languages.

**Reply by hbb20** [reply]
Yes. You certainly can. Here is wiki page that shows how to do it: https://github.com/hbb20/CountryCodePickerProject/wiki/Custom-Dialog-Title--%7C-Search-Hint-%7C-Empty-result-ACK

Please let me know if that works for you.

**Reply by astankov** [reply]
I did it, Thnak you!!!

**Comment by Umesh-01** [comment]
@hbb20 what is the current version of Country Code Picker??

**Comment by ShilpaSankaralingam** [comment]
@hbb20 Oman latest phone number shows a error.
Please Update the phone number validation for Oman.
Sample Number :
+968 77****** - > starts with 77 , 8 digit number.

**Reply by hbb20** [reply]
This library simply rely on https://github.com/MichaelRocks/libphonenumber-android for phone number validation. This will be a good point there.
